samples/nrf5340/multiprotocol_rpmsg/src/main.c

```diff
@@ -27,9 +26,9 @@
 
 #include <nrf_802154_serialization_error.h>
 
-#define LOG_LEVEL LOG_LEVEL_INFO
+#define BT_DBG_ENABLED 0
```

Comment:
Is the change in logs connected to the change in assertion handling?

Reply:
yes, the application can't define LOG_LEVEL when common/log.h is included. The LOG_LEVEL should be controlled by either BT_DBG_ENABLED or CONFIG_BT_LOG_LEVEL.
https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/bluetooth/common/log.h#L33

In order to disable the debug log as before, BT_DBG_ENABLED should be turned off explicitly.